### PR TITLE
Extended 2.3.1 (alternative)

### PR DESCRIPTION
**Pull request by daschuer**
This is a possible release candidate with the 2.3 branch merged and the CHANGELOG updated with the original 2.3.2 fixes.

### CODE REVIEW

**Comment by daschuer** [comment]
The "eslint" issue popping up in the merge is unrelated. I guess the new version is more picky.

**Comment by daschuer** [comment]
OK, so we need one to verify this and finally press merge.
Than we can add the 2.3.1 tag, update the website commit and announce our new release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 ## [2.3.1](https://launchpad.net/mixxx/+milestone/2.3.1) (Unreleased)
 
-* Added mapping for the Numark DJ2GO2 Touch controller [#4108](https://github.com/mixxxdj/mixxx/pull/4108)
+* Added mapping for the Numark DJ2GO2 Touch controller [#4108](https://github.com/mixxxdj/mixxx/pull/4108) [#4287](https://github.com/mixxxdj/mixxx/pull/4287)
 * Added mapping for the Numark Mixtrack Pro FX controller [#4160](https://github.com/mixxxdj/mixxx/pull/4160)
 * Updated mapping for Behringer DDM4000 mixer [#4262](https://github.com/mixxxdj/mixxx/pull/4262)
 * Updated mapping for Denon MC7000 controller [#4021](https://github.com/mixxxdj/mixxx/pull/4021)
+* Hercules Inpulse 300: Add better FX controls and other minor improvemrnts [#4246](https://github.com/mixxxdj/mixxx/pull/4246)
```
Comment (ronso0) [inline CHANGELOG.md:9]:
```suggestion
* Hercules Inpulse 300: Add better FX controls and other minor improvements [#4246](https://github.com/mixxxdj/mixxx/pull/4246)
```

**Comment by ronso0** [comment]
Besides the typo: LGTM
thank you for your work!

**Pull request merged**